http-clients/src/main/java/com/palantir/remoting/http/FeignClientFactory.java

```diff
@@ -216,6 +225,21 @@ public Client apply(Optional<SSLSocketFactory> sslSocketFactory) {
                     if (sslSocketFactory.isPresent()) {
                         client.sslSocketFactory(sslSocketFactory.get());
                     }
+
+                    // Set up Zipkin/Brave tracing
+                    ClientTracer tracer = ClientTracer.builder()
```

Comment:
I would also say you should have a constructor that takes the ClientTracer, people might need to tweak things.